server/container_attach.go

```diff
@@ -59,7 +59,7 @@ func (ss streamService) Attach(containerID string, inputStream io.Reader, output
 	}
 
 	controlPath := filepath.Join(c.BundlePath(), "ctl")
-	controlFile, err := os.OpenFile(controlPath, unix.O_WRONLY, 0)
+	controlFile, err := os.OpenFile(controlPath, syscall.O_WRONLY, 0)
```

Comment:
No need to backslide to the [frozen syscall package][1].  We can use [`os.O_WRONLY`][2] here.

[1]: https://golang.org/pkg/syscall/#pkg-overview
[2]: https://golang.org/pkg/os/#pkg-constants

Reply:
alrighty.